src/compute/boolean.rs

```diff
@@ -185,3 +185,39 @@ pub fn or_scalar(array: &BooleanArray, scalar: &BooleanScalar) -> BooleanArray {
         None => BooleanArray::new_null(DataType::Boolean, array.len()),
     }
 }
+
+/// Check if any of the values in the array is `true`
+pub fn any(array: &BooleanArray) -> bool {
+    if array.is_empty() {
+        false
+    } else if array.validity().is_some() {
+        array.into_iter().any(|v| v == Some(true))
+    } else {
+        let vals = array.values();
```

Comment:
Here, we need to compute if any entry is set. isn't that equivalent to `array.values().null_count() > 0`?

Reply:
Indeed. But this function returns on first `true`  value that is found, whereas `null_count` needs to scan the whole array.

Reply:
`null_count` is cached (yes, we also cache it for the values of the `BooleanArray`)

Reply:
For every bitmap? Ooooh.. then that is definitely a fast path. :)